### PR TITLE
Signup page linked to send request to backend

### PR DESCRIPTION
**Pull request by AgnesCroteau**
Link sign up page to a request that will write a new account in the database

### CODE REVIEW

**Comment by cnfrancis** [comment]
The backend barely change, front end seems good to go.

File: backend/server.js [hunk 1]
```diff
@@ -2,13 +2,15 @@ const { MongoClient, ServerApiVersion } = require('mongodb');
 const uri = "mongodb+srv://dev:hCoYCjUTGFKEC2IK@boreal.vaa1q.mongodb.net/boreal_db?retryWrites=true&w=majority";
 
 const express = require('express');
+const cors = require('cors')
```
Comment (nolanbastien) [inline backend/server.js:5]: This is a duplicate of line 7.

Remove it ! :P

File: backend/server.js [hunk 2]
```diff
@@ -2,13 +2,15 @@ const { MongoClient, ServerApiVersion } = require('mongodb');
...
 const app = express();
 const cors = require('cors');
 
 // use it before all route definitions
 app.use(cors({origin: 'http://localhost:3000'}));
 
 app.use(express.json());
+app.use(cors());
```
Comment (nolanbastien) [inline backend/server.js:13]: This is a duplicate of line 10.

File: frontend/src/Components/Pages/SignUpPage.js [hunk 3]
```diff
@@ -1,10 +1,47 @@
...
+                }),
+            });
+            let resJson = await res.json();
+            if (res.status === 200) {
+                setName("");
+                setEmail("");
+                setPassword("");
+                setAccountType("Customer");
+
+                // TODO: User created successfully, redirect to correct page here
```
Comment (nolanbastien) [inline frontend/src/Components/Pages/SignUpPage.js:36]: To redirect to home page you can use the same function as on line 48: 

`navigate('/')`

**Review by nolanbastien — CHANGES_REQUESTED** [review]
Good work Agnès ! :)

**Review by cnfrancis — APPROVED** [review]

**Pull request merged**